src/main/kotlin/step2/StringCalculator.kt

```diff
@@ -14,16 +14,11 @@ class StringCalculator(
         parseInput(input)
     }
 
-    fun calculate(): Int {
-        var calculated = operands.first()
-
-        for (index in operators.indices) {
-            calculated = operators[index].operate(calculated, operands[index + 1])
+    fun calculate() =
```

Comment:
I always find it confusing when to use single-expression functions versus block bodies. Generally, I tend to use single-expression functions when the function is relatively unimportant and can be written as a single line (or with method chaining).

I've noticed that many developers use single-expression functions whenever they can fit them on one line. However, I feel that doing so slightly reduces readability, as it makes the function look like a simple getter or setter. 

I'm just genuinely curious about what criteria you use to distinguish between the two.